src/Umbraco.Core/Services/PropertyValidationService.cs

```diff
@@ -48,13 +51,35 @@ public PropertyValidationService(
         ICultureDictionary cultureDictionary,
         ILanguageService languageService,
         IOptions<ContentSettings> contentSettings)
+        : this(
+            propertyEditors,
+            dataTypeService,
+            textService,
+            valueEditorCache,
+            cultureDictionary,
+            languageService,
+            contentSettings,
+            StaticServiceProvider.Instance.GetRequiredService<IDataValueEditorFactory>())
+    {
+    }
+
+    public PropertyValidationService(
+        PropertyEditorCollection propertyEditors,
+        IDataTypeService dataTypeService,
+        ILocalizedTextService textService,
+        IValueEditorCache valueEditorCache,
+        ICultureDictionary cultureDictionary,
+        ILanguageService languageService,
+        IOptions<ContentSettings> contentSettings,
+        IDataValueEditorFactory dataValueEditorFactory)
     {
         _propertyEditors = propertyEditors;
         _dataTypeService = dataTypeService;
         _textService = textService;
         _valueEditorCache = valueEditorCache;
         _cultureDictionary = cultureDictionary;
         _languageService = languageService;
+        _dataValueEditorFactory = dataValueEditorFactory;
```

Comment:
If you agree with the comments made below, the `IDataValueEditorFactory` is no longer necessary, which means the new constructor can be removed.